Cargo.toml

```diff
@@ -10,15 +10,109 @@ members = [
     "packages/ic-response-verification",
     "packages/ic-response-verification-test-utils",
     "packages/ic-response-verification-tests",
+    "packages/ic-response-verification-wasm",
+    "packages/ic-certification-testing-wasm",
 ]
 
 # https://github.com/rust-lang/cargo/issues/9406
-# these projects need wasm32-unknown-unknown as their target
-exclude = [
-    "packages/ic-response-verification-wasm",
-    "packages/ic-certification-testing-wasm",
+# includes all members except those that must be compiled to WASM
+default-members = [
+    "examples/rust",
+    "packages/ic-cbor",
+    "packages/ic-certification",
+    "packages/ic-certificate-verification",
+    "packages/ic-certification-testing",
+    "packages/ic-representation-independent-hash",
+    "packages/ic-response-verification",
+    "packages/ic-response-verification-test-utils",
+    "packages/ic-response-verification-tests",
 ]
 
+[workspace.package]
+version = "1.2.0"
+authors = ["DFINITY Stiftung"]
+edition = "2021"
+repository = "https://github.com/dfinity/response-verification"
+homepage = "https://github.com/dfinity/response-verification#readme"
+license = "Apache-2.0"
+
+[workspace.dependencies]
+nom = "7.1"
+leb128 = "0.2"
+log = "0.4"
+hex = "0.4"
+base64 = "0.21"
+http = "0.2"
+flate2 = "1.0"
+sha2 = "0.10"
+urlencoding = "2.1.3"
+rstest = "0.18"
+tokio = { version = "1.24", features = ["full"] }
+
+serde_bytes = "0.11"
+serde_cbor = "0.11"
+
+thiserror = "1.0"
+anyhow = "1.0"
+
+candid = "0.9"
+ic-agent = "0.29"
+ic-utils = "0.29"
+ic-cdk = "0.6.0"
+ic-cdk-macros = "0.6.0"
+
+wasm-bindgen = "0.2"
+wasm-bindgen-test = "0.3"
+serde-wasm-bindgen = "0.5"
+js-sys = "0.3"
+console_error_panic_hook = "0.1"
+wasm-bindgen-console-logger = "0.1"
+# https://github.com/rust-random/rand#wasm-support
+# https://docs.rs/getrandom/latest/getrandom/#webassembly-support
+rand = "0.8"
+getrandom = { version = "0.2", features = ["js"] }
+
+
+ic-certification = { path = "./packages/ic-certification", default-features = false, version = "1.2.0" }
```

Comment:
Unfortunately I couldn't find a way to not manually specify these versions. Without them Cargo complains when trying to publish and I can't figure out how to reference `workspace.package.version` here :(

Reply:
For those local crates that have their own `Cargo.toml` ins't cargo taking that version if you omit here? 🤔

Reply:
If `ic-certification` specifies it's version as `workspace.version` then that's the version it will take when publishing to crates.io, but it will not read the version of `ic-certification`'s dependencies from their `Cargo.toml`.

For example, when publishing `ic-response-verification` which depends on `ic-certification`, it will read `ic-response-verification`'s `Cargo.toml` but not `ic-certification`'s `Cargo.toml` when publishing. That's why the version also needs to be specified manually for the dependencies.